.github/workflows/ci.yml

```diff
@@ -81,39 +81,28 @@ jobs:
           - machine: ubuntu-20.04
             distribution: linux-glibc
             log-dir: "/var/log/opentelemetry/dotnet"
-          - machine: ubuntu-20.04
-            distribution: linux-musl
-            test-instrument: "false"
           - machine: macos-11
             distribution: macos
             log-dir: "/var/log/opentelemetry/dotnet"
     runs-on: ${{ matrix.machine }}
     steps:
-      - run: brew install coreutils
-        if: ${{ runner.os == 'macOS' }}
-      - name: Download OTel .NET AutoInstrumentation using download.sh
-        shell: bash
-        run: |
-          set -o pipefail
-          curl -sSfL https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/download.sh |
-            DISTRIBUTION=${{ matrix.distribution }} bash -s
-      - name: Test download.sh - verify if binaries are downloaded
-        shell: bash
-        run: test "$(ls -A ./otel-dotnet-auto)"
       - uses: actions/setup-dotnet@v2.1.1
         with:
           dotnet-version: 6.0.x
-      - run: sudo mkdir -p ${{ matrix.log-dir }} && sudo chmod a+rwx ${{ matrix.log-dir }}  # workaround before next release
-        if: ${{ runner.os != 'Windows' && matrix.test-instrument != 'false' }}
-      - name: Instrument dotnet using instrument.sh
+      - run: brew install coreutils
+        if: ${{ runner.os == 'macOS' }}
+      - name: Create test directory
+        run: sudo mkdir -p ${{ matrix.log-dir }} && sudo chmod a+rwx ${{ matrix.log-dir }}  # workaround before next release
+        if: ${{ runner.os != 'Windows' }}
+      - name: Test the Shell scripts from README.md
         shell: bash
         run: |
           set -e
-          curl -fL https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/instrument.sh -O
-          DISTRIBUTION=${{ matrix.distribution }} source ./instrument.sh
+          export DISTRIBUTION=${{ matrix.distribution }} 
+          curl -sSfL https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/download.sh -O
+          sh ./download.sh
+            test "$(ls -A ./otel-dotnet-auto)"
+          curl -sSfL https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/instrument.sh -O
+          source ./instrument.sh
```

Comment:
What do you think of including the `instrument.sh` and `instrument.ps1` to the `zip` file (so that it does not have to be additionally downloaded)?

Then we could change these 2 lines to 

```sh
source ./otel-dotnet-auto/instrument.sh
```

I would rather address it in a separate PR.